packages/http-framework/src/lib/interactions/utils/util.ts

```diff
@@ -36,7 +37,37 @@ export async function runner<Type extends InteractionType, Data>(
 		return handleError(reply, error);
 	}
 
-	return isGeneratorObject(result) ? handleGenerator(reply, interaction, result) : reply.status(HttpCodes.OK).send(result);
+	return isGeneratorObject(result) ? handleGenerator(reply, interaction, result) : handleResponse(reply, result);
+}
+
+function hasFiles(data: NonNullObject): data is { files: RawFile[] } {
+	return Array.isArray((data as any).files) && (data as any).files.length > 0;
```

Comment:
why as any?

Reply:
To access unknown properties, `NonNullObject` doesn't allow to read properties